src/test/crimson/seastore/nvmedevice/test_nvmedevice.cc

```diff
@@ -57,6 +57,7 @@ TEST_F(nvdev_test_t, write_and_verify_test)
   run_async([this] {
     device.reset(new random_block_device::nvme::NVMeBlockDevice(""));
     device->open(dev_path, seastar::open_flags::rw).unsafe_get();
+    device->set_block_size(BLK_SIZE);
```

Comment:
It doesn't seem reasonable to set an external BLK_SIZE, as the normal path always set it from the device.

It might be more reasonable to go through the device-level procedure (mkfs/mount) instead of relying on the internal interfaces. So in the best case we can have no specializations for tests and with full coverage.

Probably it also means that some config logic will need to be moved out of the RBMDevice, for example to set the cbjournal size from the RBMDevice constructors instead.

What do you think?

Reply:
Seems reasonable.